tools/staramr/staramr_search.xml

```diff
@@ -89,11 +95,20 @@
                 </when>
                 <when value="none" />
             </conditional>
+            <conditional name="plasmidfinder_type">
+                <param name="plasmidfinder_type_condition" type="select" label="Specify PlasmidFinder Database type">
+                    <option value="default" selected="true">Include all available database types</option>
```

Comment:
Same here. Instead of `default` could you use something more descriptive like `include_all`?

Reply:
Yeah changed it to `include_all` in my next commit